packages/server/src/migrations/set_next_payment_for_subscriptions.ts

```diff
@@ -0,0 +1,35 @@
+import { Migration } from '@mikro-orm/migrations';
+
+type Subscription = {
+  _id: string;
+  anchorDate: Date;
+  nextPayment: Date;
+};
+
+export class MigrationSetNextPaymentForSubscriptions extends Migration {
+  async up(): Promise<void> {
+    await this.ctx?.schema.alterTable('subscription', (table) => {
+      table.date('nextPayment').defaultTo(new Date());
+    });
+
+    const subscriptions = await this.ctx?.table('subscription').where({ status: 'active' }).select<Subscription[]>();
+    for await (const subscription of subscriptions || []) {
+      const nextPayment = new Date(); // TODO: think about fair nextPayment date and how we can calculate it
```

Comment:
@mariusheine Do you have an idea to what we can set the `nextPayment` dates? Correct would be setting it to the date of the latest draft invoice, however some (actually almost all 🙈 ) of the subscriptions in our deployment are stuck as there are no draft invoices atm, so we would need to fix all of them by hand.

Reply:
Using latest draft invoice now and set an error otherwise.